exotica_core/CMakeLists.txt

```diff
@@ -23,7 +23,13 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 find_package(TinyXML2 REQUIRED)
 find_package(ZeroMQ REQUIRED)
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(MSGPACK msgpack)
+pkg_check_modules(MSGPACK QUIET msgpack)
```

Comment:
Did we actually test and verify that Meshcat is broken on Kinetic or are we just deactivating it because the zeromq didnt have the pkg-config?

Reply:
I interpreted Vlad's comment as if the version in Ubuntu 16.04 is incompatible with the codebase.

Reply:
I compiled it on 16.04. It compiles with warning but the features are not available in msgpack so I disabled it in the code but left it in CMAKE so that we don't have to propagate that logic to exotica_python and the examples. They will just fail gracefully when run.